lib/i18n/tasks/translators/openai_translator.rb

```diff
@@ -66,14 +83,11 @@ def translate_values(list, from:, to:)
       results.flatten
     end
 
-    def translate(values, from, to) # rubocop:disable Metrics/MethodLength
+    def translate(values, from, to)
       messages = [
         {
           role: 'system',
-          content: "You are a helpful assistant that translates content from the #{from} to #{to} locale in an i18n
-          locale array. The array has a structured format and contains multiple strings. Your task is to translate
-          each of these strings and create a new array with the translated strings. Keep in mind the context of all
-          the strings for a more accurate translation.\n"
+          content: system_prompt.gsub('{{from}}', from).gsub('{{to}}', to)
```

Comment:
I would have loved using `Kernel#format` rather than `gsub` here, but the presence of `%{ and ending with }` in the default prompt prevents it: `format` considers it an interpolation named ` and ending with ` and raises an error because… well, we don't provide a value for such a variable. :sweat_smile:

Reply:
You can use `%%` to escape a percent in a format string

https://github.com/ruby/ruby/blob/46ef74f270d022068def440ab6208e1648030847/sprintf.c#L436-L440

Reply:
Nice, thanks: I just changed it :ok_hand: